src/Proto.Cluster/DefaultClusterContext.cs

```diff
@@ -202,7 +202,6 @@ IFuture future
 
             if (!context.System.Shutdown.IsCancellationRequested)
                 if (Logger.IsEnabled(LogLevel.Debug)) Logger.LogDebug("TryRequestAsync timed out, PID from {Source}", source);
-            _pidCache.RemoveByVal(clusterIdentity, pid);
```

Comment:
What is the effect of removing this? looks like we could get stuck with an invalid pid in the pid cache, or is it removed elsewhere now?

Reply:
This change was commited by mistake, reverting.